### ISSUE TITLE
GL/Vulkan: Game that need DMA detiling.

### ISSUE THREAD

**Issue by Triticum0** [RPCS3/rpcs3#13680]
- mercury hg
- Polar Panic  #6609
- Star Wars the force unleashed #12756
- Star Wars the force unleashed  II #8244
- Retro/Grade #6688
- The Chronicles of Riddick Assault on Dark Athena

**Comment by RokkuDayo** [comment]
The Witch and The Hundred Knight has the same issue as well.

**Comment by poudink** [comment]
Mercury Hg seems to need at least Write Color Buffers, Write Depth Buffers, Read Color Buffers and Force CPU Blit, but it's still completely broken. Handle RSX memory tiling doesn't help at all.

![image](https://github.com/RPCS3/rpcs3/assets/29778535/c6d0baf6-e003-4cc5-bbb3-97b85450a0ac)

**Comment by elad335** [comment]
@kd-11 anything left here?

**Comment by Triticum0** [comment]
Can't test mercury hg anymore as crashes the emulator for me now

**Comment by Triticum0** [comment]
@elad335 Yes, mercury hg and The Chronicles of Riddick Assault on Dark Athena still have graphical issues, even with Handle RSX memory tiling is enable could be unrelated but both have been tested recently 

 mercury hg

<img width="1737" height="1066" alt="Image" src="https://github.com/user-attachments/assets/9e2b1fd2-8036-4f6c-af4d-c9263881a139" />

The Chronicles of Riddick Assault on Dark Athena

<img width="1280" height="720" alt="Image" src="https://github.com/user-attachments/assets/e10ec750-9452-4576-98bb-e5a7890b61a7" />

**Comment by kd-11** [comment]
There's some unimplemented edge case in some games.
